src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,46 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("Set에 담긴 값의 갯수를 반환한다")
+    @Test
+    void testSize() {
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @DisplayName("Set에 담긴 값이 있으면 true를 반환한다")
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    void testContainsWithValueSource(Integer value) {
+        assertThat(numbers.contains(value)).isTrue();
+    }
+
+    @DisplayName("Set에 담긴 값이 있으면 true를 반환한다")
+    @ParameterizedTest
+    @CsvSource(value = {"1:true", "2:true", "3:true", "4:false", "5:false"}, delimiter = ':')
+    void testContainsWithCsvSource(String input, String expected) {
```

Comment:
```suggestion
    void testContainsWithCsvSource(int input, boolean expected) {
```
파싱하실 필요 없이, 이렇게 사용도 가능합니다~!

Reply:
코멘트 감사합니다.~
해당 기능 처음 접했는데 너무 좋은 것 같습니다.